Massive inhand sprite changes

MitchTheUser opened the pull request:
<!-- The text between the arrows are comments - they won't be visible on your PR. -->
<!-- To label this PR, add the label(s) without the prefixes surrounded by brackets anywhere, for example: [LABEL] -->
<!-- PRs should at least have one area (A-) label and at least one category (C-) label -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->

Over 70 items have had their existing inhand sprites improved upon. These vary from simple recolors, to design changes and major improvements in the case that its item has been completely redesigned. 

The positions of a few of these inhands have also been changed when it made them look nicer, such as improving the north facing sprites for some of them, instead of having the character awkwardly hold the item behind their back. 

Various matsci inhands like metal/glass sheets, tiles, rods and cable coils also now use grayscale inhand sprite colors, which makes them look nicer when the material color of their icon sprites are overlayed. 

A list of most of the inhand sprite changes is below, though the actual amount changed is likely higher.
![image](https://github.com/goonstation/goonstation/assets/94775877/4e50c23b-482a-4eb5-b552-a074bde28424)

## Why's this needed? <!-- Describe why you think this should be added to the game. -->

Inhand sprites have received very little polish over the years in response to resprites, yet they are still something you see every time you hold an item, and deserve some care and polish. 

There are many more inhands that are still old and in need of improvement, but this PR should help to modernize a lot of them to make more inhands match the look of the new sprites they are based on.

## Changelog <!-- If necessary, put your changelog entry below. Otherwise, /please/ delete this entire section. -->
<!-- Put how you want to be credited in the changelog in place of CodeDude. -->
<!-- Use (*) for major changes and (+) for minor changes. See the contributor guide for details. For example: -->

```changelog
(u)MitchSmith
(*)A large amount of inhands have been improved.
```

flappybatpal reviewed (commented):
Missing:
 `/obj/item/reagent_containers/food/snacks/donut` which is made by the oven does not use the donut inhand.
No wrench subtypes (yellow, green etc) have inhands.
`/obj/item/furniture_parts/rack`
`/obj/item/tank/emergency_oxygen`



Missing right hand west facing inhands?
Knife (also no left hand east)
Meatball (also no left hand east)
Burger (also no left hand east)
Donut (also no left hand east)
Bike horn
Vuvuzela  (also no left hand east) 
Syringe
Table parts
Metal sheet (also no left hand east)
Glass sheet (also no left hand east)

All books use the paper sprite, while there is significant cosmetic difference between a lot of books I would have expected these to use something more like the bible.

Golden (spare) ID card is quite hard to see in hand.
Burger is rather difficult to identify.
Saxophone does not look very different to previous?

MitchTheUser commented:
> 

Fixed the various items not displaying their inhands, also fixed a few more like the hand labeler. 

Added missing inhand directions, the originals never had inhands for those directions but they have them now. I don't know why the syringe was on the list, I checked the branch and it has right hand facing west inhands, it's just that only the black needle is visible.

All books should now use a separate generic looking 'book' inhand instead of paper, and paper continues to use the 'paper' inhand.

Golden ID was a bit tricky, I made it a bit brighter, hopefully it's a bit easier to see now? The original items colors weren't cutting it for the inhands visibility

Burger has a completely new inhand, now you hold the entire burger instead of just the bun being visible (tasty).

Saxophone inhand change has been reverted, I can't remember what I was going for since it was a change early in the project, but I agree that it is quite minor, so now it will remain the same inhand that is currently in the game.

In addition, I made some tiny tweaks to a couple inhands (Frosted donut outline, 1 pixel on wrench) and created brand new inhands for the cinnamon, robust, and robusted donut sprites (yummy). All donuts should use their proper inhands.

flappybatpal commented:
There might be requests for tweaks but this seems like an overall improvement and has been open a month.
